src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -953,7 +954,7 @@ verify_request_signature(Req,
              State#base_state{log_msg = {not_found, user_or_client}}};
         {error, Error} ->
             Msg = verify_request_message(error_finding_user_or_client, Name, OrgName),
-            {false, wrq:set_resp_body(chef_json:encode(Msg), Req),
+            {{halt, 500}, wrq:set_resp_body(chef_json:encode(Msg), Req),
```

Comment:
That sounds good to me. I was just whipping something up really quick without tests (and even dialyzer specs) to see if it would pass pedant (it did). I deferred the decision on 500/503 because there are so many other places where we could catch the `no_connections` and return 503. Having said that, it's not blocking us from doing so here.